docs/gcp/quick-start/cluster-management.mdx

```diff
@@ -6,7 +6,7 @@ image: "https://kubefirst.konstruct.io/docs/img/kubefirst.svg"
 ---
 
 :::note
-The MVP release of the new Cluster Management feature for Google Cloud only includes Virtual clusters. Physical clusters added in a patch release in the coming days.
+The MVP release of the new Cluster Management feature for Google Cloud only includes Virtual clusters for GitLab users. Physical clusters added in a patch release in the coming days.
```

Comment:
I would leave out the bit about physical clusters if this is an update that matters for only a week or so, unless I'm misunderstanding?

Reply:
Have removed the second sentence